### DISCUSSION TITLE
*** 좀비 헌터  프로토타입 완성 ***

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#557] [4. General]
안녕하세요 일등하이 입니다 

그동안 제페토 게임을 만들어 보려고 몇일간 커뮤니티 오가며 만들어봤습니다 

정말 커뮤니티 답변 주시는 분들 덕분에 프로토 타입을 완성 할수 있었습니다 감사합니다 : )

좀비 헌터는 멀티플레이 슈팅 액션장르 게임이며 플레이어들과 협업하여 좀비를 물리쳐 성장하고 유저들끼리 토너먼트로 싸울수 있는 공간을 제공합니다 

아직 프로토타입 단계이지만 여기서 멀티플레이도 붙이고 재미 요소도 넣어서 꼭 서비스 해보고 싶네요 

다시한번 답변주신 모든 분들 정말 감사합니다 

앞으로 더욱 많이 물어볼게요 ~ :) 

https://user-images.githubusercontent.com/50188795/160803681-847c22d8-7bdd-4197-b3d0-18cb18b32761.mp4

**Comment by x4cf** [comment]
우와! 이런 멋진 프로젝트가 있다는걸 지금 보았네요! 멋진걸요!
